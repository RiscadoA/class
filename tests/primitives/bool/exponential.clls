proc main() {
    cut {
        let! x false
        |x: ~!lbool|
        ?x;
        println(x);
        println(x and x);
        println(!x);
        println(x or x);
        if x {
            println("x is true");
            ()
        } else {
            println("x is false");
            ()
        }
    }
};;
