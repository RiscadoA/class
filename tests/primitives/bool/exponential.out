false
false
true
false
x is false
